src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
Ok, the session stuff can be discussed in the other PR, but it doesn't belong with the Data-JSON removal